fill/actuators/qd.cc

```diff
@@ -1,19 +1,32 @@
 #include "qd.h"
 
-QD::QD(){
-
+QD::QD() {
+    stepperMotor = new StepperMotor(COIL1, COIL3);
 }
 
 QD::~QD(){
-
+    delete stepperMotor;
 }
 
 bool QD::Actuate(){
-    return true; 
+    if (actuated){
+        return false; // sensor did not actuate 
+    }
+
+    stepperMotor->Enable(); // enable sensor motor 
+    stepperMotor->Rotate(stepperMotor->CLOCKWISE, 50); // Rotate the motor clockwise, 50ms time delay
+
+    delay(60000); // We need 1200 steps
```

Comment:
We should probably discuss how we want to structure the main control loop more. Using delay will only be acceptable if we start actuation in a separate thread since we cannot block the telemetry functionality

Reply:
Not a blocker for this PR though